DbAdapterTestUtilities/Stubs/StubFieldValue.h

```diff
@@ -26,39 +21,39 @@ namespace systelab { namespace db { namespace test_utility {
 		StubFieldValue(const std::string& name, bool value);
 		StubFieldValue(const std::string& name, double value);
 		StubFieldValue(const std::string& name, const std::string& value);
```

Comment:
Based on my experience with this class, there is a problem that might be solved with the use **explicit** keyword despite being two arguments, because we have alternative constructors based on type of the second param only.

The problem was stuff like `StubFieldValue("MyField","theValue")`. Since you are using a const chart* literal and there is no specific constructor for that, that type needs to be implicitly converted. One would think that the more logical conversion would be to string, but what happens is that as it is a pointer it can be implicitly converted to bool, and it gets the bool constructor.
Potential fixes:
* Maybe if the definition order had string before bool that would be the chosen option.
* Using explicit will force you to stick to one of those types; so you could not do that and would require to construct a string, though could use the string literal.
* Having a specific constructor receiving a const char*, which would be the chosen option then

Reply:
Interesting, I didn't noticed that pitfall. I'll set it as explicit. No matter that the client has to do some extra work, it is what it is